app/components/WoWResults/ShortagePredictor/Results.tsx

```
@@ -184,10 +116,21 @@ const PredictionTable = ({
-      header: 'Oribos Link',
+      header: 'Undermine',
       accessor: ({ row }) => OribosLink({ row: { itemID: row.item_id } })
     },
-    { columnId: 'quality', header: 'Quality' },
+    {
+      columnId: 'current_price_vs_avg_percent',
+      header: 'Price vs Avg %',
+      accessor: ({ getValue }) => {
+        const value = getValue()
+        if (typeof value === 'string') {
+          return <p>{parseFloat(value).toFixed(2)}%</p>
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)